src/d_main.c

```diff
@@ -1493,7 +1496,10 @@ static void AutoloadPWadDir(void (*AutoLoadFunc)(const char *path))
         for (int j = 0; j < array_size(autoload_paths); ++j)
         {
             char *dir;
-            if (!strncasecmp(M_BaseName(wadfiles[i]), "rekkr", 5)) {
+            if (!strncasecmp(M_BaseName(wadfiles[i]), "rekkr.wad", 9)
+                || !strncasecmp(M_BaseName(wadfiles[i]), "rekkrs1.wad", 11)
+                || !strncasecmp(M_BaseName(wadfiles[i]), "rekkrsl.wad", 11))
+            {
```

Comment:
Please change the `GetAutoloadDir(..., true);` to `false`.

Ans please move opening brackets to the next line while at it.

Reply:
Yep, right, sorry for the noise!